internal/registry/registry_test_helper.go

```diff
@@ -0,0 +1,66 @@
+package registry
```

Comment:
Should this file be named `registry_helper_test.go` so it's skipped when compiling the main binary?

Reply:
it will be compiled but no linked, but i will rename it anyway